### PR TITLE
tests: update chromium installable source path

### PR DESCRIPTION
**Pull request by paulirish**
This is a minor followup from #11763. just a tweak to avoid a dumb 404.

------- 

also, spoiler alert: the issues panel test fails

image

but i (or someone else) will have to deal with that separately.

### CODE REVIEW

File: third-party/chromium-synchronization/installability-errors-test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('installabilityErrors', () => {
 
   beforeAll(async () => {
     const installableLoggingGitTilesUrl =
-      'https://chromium.googlesource.com/chromium/src/+/master/components/webapps/installable/installable_logging.cc?format=TEXT';
+      'https://chromium.googlesource.com/chromium/src/+/master/components/webapps/browser/installable/installable_logging.cc?format=TEXT';
```
Comment (patrickhulce) [inline third-party/chromium-synchronization/installability-errors-test.js:19]: file moved?

**Review by patrickhulce — APPROVED** [review]
does anyone get notified about these failures? I would expect to see some runs under "Markdown Links Checker" (oof name for these chromium checks), but I see 0 runs.

https://github.com/GoogleChrome/lighthouse/actions/workflows/cron-weekly.yml

**Comment by paulirish** [comment]
> does anyone get notified about these failures? I would expect to see some runs under "Markdown Links Checker" (oof name for these chromium checks), but I see 0 runs.

heh. the big changes to https://github.com/GoogleChrome/lighthouse/blob/master/.github/workflows/cron-weekly.yml just landed yesterday.. so its 'weekly' schedule hasnt been hit yet.

File: third-party/chromium-synchronization/installability-errors-test.js (hunk 1 above)
Comment (paulirish) [reply third-party/chromium-synchronization/installability-errors-test.js:19]: ayup

**Comment by paulirish** [comment]
> > does anyone get notified about these failures? 

tbh i'm not sure about notifications. i get some cron-ish gha notification on my other projects, but i'm not sure how thing will work with this.

they should def be visible in the 'Actions' tab after it runs tho.

**Pull request merged**